addons/behaviour_toolkit/finite_state_machine/fsm.gd

```diff
@@ -51,16 +51,19 @@ var current_events: Array[String]
 var current_bt_status: BTBehaviour.BTStatus
 
 
-func _ready() -> void:
-	connect("state_changed", _on_state_changed)
 
+# Connecting signal using @onready to omit the need to use super() call
+# in _ready() of extended nodes if they override _ready().
+@onready var __connect_child_order_changed: int = \
+	child_order_changed.connect(_update_states_list)
```

Comment:
I am not a fan of using `\` to wrap code in such case as I think it makes the code very messy tbh

Reply:
Same, but what other choice we have? If I make it shorter it will still be to long or cryptic. Scrolling way right is also not very nice solution. I made it like this because it's the lesser evil but I'm open for suggestions.

Reply:
Are you using the Godot Editor? In VSCode this line is easily fully visible and doesn't need any scrolling.

Reply:
> Are you using the Godot Editor? In VSCode this line is easily fully visible and doesn't need any scrolling.

Ah, no, no, I have a lot of space to see even very long lines past "100 characters line" even xD (most of the time):
![image](https://github.com/ThePat02/BehaviourToolkit/assets/6152134/7b634795-76ee-49a9-84d8-c043e6efc03b)

I just break lines so they don't have more thatn 80 characters like PEP8 demands it xD
It is usefull tho, when you want to work in split file view or make editor snap to half of your monitor.

I can follow different rules I just did what is usually done in python files. Lesser evil although I got used to it too.

Reply:
Drive-by: I'd recommend enforcing use of this [formatter](https://github.com/Scony/godot-gdscript-toolkit_

Reply:
> Drive-by: I'd recommend enforcing use of this [formatter](https://github.com/Scony/godot-gdscript-toolkit)

Oh! This cool I will use it for my projects for sure, thx for the link!

Reply:
> Drive-by: I'd recommend enforcing use of this [formatter](https://github.com/Scony/godot-gdscript-toolkit)

Works like a charm. I'll use it after most 2.0 features are merged. Maybe I'll set up some github actions to do it too!